spec/features/crops/crop_wrangling_button_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+feature "crop wrangling button" do
+
+  let(:crop_wrangler_member) { FactoryGirl.create(:crop_wrangler_member) }
+
+    context "crop wrangling button" do
+
+      background do
+        login_as(:crop_wrangler_member)
+      end
+
+      background do
+        visit crops_path
+      end
```

Comment:
Got it. The spec/factories/member.rb file does say "crop_wrangling_member." Thank you for pointing me there!